bool[] fleet_dock_status_array
string[] fleet_names_array